pennylane/estimator/templates/select.py

```diff
@@ -286,8 +335,8 @@ def resource_decomp(
         gate_list.append(GateCount(s, 2 * num_orb))
         gate_list.append(GateCount(s_dagg, 2 * num_orb))
 
-        # Adjoint of QROM for one body integrals Eq. 35 in arXiv:2011.03494
-        gate_list.append(GateCount(resource_rep(qre.Adjoint, {"base_cmpr_op": qrom_onebody})))
+        # Adjoint of QROM for two body integrals only Eq. 35 in arXiv:2011.03494
+        gate_list.append(GateCount(resource_rep(qre.Adjoint, {"base_cmpr_op": qrom_twobody})))
```

Comment:
Please be careful with this changes. Currently, it is confusing why the changes are made.